uint32 Pepsi_x
uint32 Pepsi_y

uint32 DietMTD_x
uint32 DietMTD_y

uint32 Coke_x
uint32 Coke_y
